### PR TITLE
fix#26

### PR DESCRIPTION
**Pull request by Friedrich-Mocker**

### CODE REVIEW

**Comment by ain** [comment]
For the reference, please also see http://conventions.interactive-pioneers.com/VCS#commits

**Comment by GeorgMeyer23** [comment]
do not add distribution files except for release commits

File: src/iptools-jquery-modal.js [hunk 1]
```diff
@@ -254,6 +254,7 @@
       if (!$overlay) {
         $overlay = $('<div/>')
           .addClass(settings.overlayClass)
+            .css('z-index', (settings.zIndex - 2))
```
Comment (GeorgMeyer23) [inline src/iptools-jquery-modal.js]: wrong indentation

File: src/iptools-jquery-modal.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@ $IPTModal_animation-speed: 0.25s;
   position: fixed;
   top: 0;
   left: 0;
-  z-index: 100;
+
```
Comment (GeorgMeyer23) [inline src/iptools-jquery-modal.scss]: remove empty line

File: src/iptools-jquery-modal.scss [hunk 3]
```diff
@@ -20,7 +20,7 @@ $IPTModal_animation-speed: 0.25s;
 
 .modal {
   position: absolute;
-  z-index: 102;
+
```
Comment (GeorgMeyer23) [inline src/iptools-jquery-modal.scss]: remove empty line

**Comment by beroes** [comment]
Checked it out, it works out of the box, no regressions found, safe to merge.

**Comment by ain** [comment]
The issues @GeorgMeyer23 pointed out have been resolved before we can merge.

**Comment by ain** [comment]
Thanks 👍

**Pull request merged**